staging/src/k8s.io/kubectl/pkg/cmd/replace/replace.go

```diff
@@ -227,6 +227,10 @@ func (o *ReplaceOptions) Validate() error {
 		return fmt.Errorf("--timeout must have --force specified")
 	}
 
+	if o.DeleteOptions.ForceDeletion && o.DryRunStrategy != cmdutil.DryRunNone {
+		return fmt.Errorf("dry-run can not be used when --force is set")
+	}
+
 	if cmdutil.IsFilenameSliceEmpty(o.DeleteOptions.FilenameOptions.Filenames, o.DeleteOptions.FilenameOptions.Kustomize) {
 		return fmt.Errorf("Must specify --filename to replace")
```

Comment:
Since you're at it, can you make the above error lowercase: 
```suggestion
		return fmt.Errorf("must specify --filename to replace")
```